src/buildwin.rst

```diff
@@ -40,41 +40,28 @@ The following feedstocks are examples of this build structure deployed:
 * `libping <https://github.com/conda-forge/libpng-feedstock/blob/master/recipe/bld.bat>`_
 * `pugixml <https://github.com/conda-forge/pugixml-feedstock/blob/master/recipe/bld.bat>`_
 
-VC features
------------
-Conda builds Windows packages with features as a means of keeping the Visual Studio version (``vc``) used to build packages uniform across a given Python version.
 
-Rule of thumb:
+Building for different VC versions
+----------------------------------
 
-* if it's a shared library that something other than Python will use, you only need python in build to activate features. You need the features block;
-* if it is a shared library that is tied to a Python API (e.g. Boost Python), you need BOTH, and the features block;
-* if it is a library that has compiled content, but is used only from Python, you need Python in both build and runtime requirements, but you do NOT need the features block.
+On Windows, different Visual C versions have different ABI and therefore a package needs to be built for different Visual C versions. Packages are tied to the VC version that they were built with and some packages have specific requirements of the VC version. For example, python 2.7 requires ``vc 9`` and python 3.5 requires ``vc 14``.
 
-\* Note that, because PY35 and PY36 use the same ``vc`` the user can skip py36 (or py35) in the first case above because the ``vc14`` package created will work on both versions.
-
-To provide features add the following lines to the build section:
+With ``conda-build 3.x``, ``vc`` can be used as a selector when using the ``compiler`` jinja syntax.
 
 .. code-block:: yaml
 
-    build:
-      features:
-        - vc9   # [win and py27]
-        - vc10  # [win and py34]
-        - vc14  # [win and py>=35]
+    requirements:
+      build:
+        - {{ compiler('cxx') }}     # [win]
```

Comment:
Why does this line have a windows selector?

Reply:
See https://github.com/conda-forge/conda-forge.github.io/pull/576/files#diff-29e8b4bf4d69b118f7b3c877f2f39478R141

Reply:
Yea sorry I read the other one first :sheep:

Reply:
Now that we have a `compiler` implementation for all platforms, should we drop the selector?